ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -199,6 +202,41 @@ public void SetPointerEvents(BorderedCanvas view, string pointerEventsValue)
             view.SetPointerEvents(pointerEvents);
         }
 
+        /// <summary>
+        /// Set accessibility traits for the view.
+        /// </summary>
+        /// <param name="view">The view.</param>
+        /// <param name="accessibilityTraitsValue">Can be <see cref="JArray"/> of objects or a single object. 
+        ///     String representation of the object(s) is parsed as <see cref="AccessibilityTrait"/>.</param>
+        [ReactProp("accessibilityTraits")]
+        public void SetAccessibilityTraits(BorderedCanvas view, object accessibilityTraitsValue)
+        {
+            AccessibilityTrait[] result = null;
+            if (accessibilityTraitsValue != null)
+            {
+                if (accessibilityTraitsValue is JArray asJArray)
+                {
+                    var list = new List<AccessibilityTrait>();
+                    foreach (var s in asJArray.Values<string>())
```

Comment:
can LINQ be used here for shorter notation?

Reply:
I wasn't able to use LINQ due to the out parameter in the TryParse method. If you know how please show an example.

---
In reply to: [168846666](https://github.com/gmangov/react-native-windows/pull/1#discussion_r168846666) [](ancestors = 168846666)